cmd/detectExecuteScan.go

```diff
@@ -46,6 +46,12 @@ func addDetectArgs(args []string, config detectExecuteScanOptions) []string {
 
 	args = append(args, fmt.Sprintf("--detect.project.name=%v", config.ProjectName))
 	args = append(args, fmt.Sprintf("--detect.project.version.name=%v", config.ProjectVersion))
+	args = append(args, fmt.Sprintf("--detect.policy.check.fail.on.severities=%v", config.FailOn))
```

Comment:
currently it can be empty if `failOn` is configured incorrectly.
I see two options:
* make the parameter mandatory
* only add the flag if `failOn` is set

What do you think?

In addtion we should do something like `strings.Join(config.FailOn, ",")` if the type is changed to`[]string`